src/psyclone/f2pygen.py

```diff
@@ -37,8 +37,7 @@
     provide routines which can be used to generate fortran code. This library
     includes pytest tests. '''
 
-import fparser
-import fparser.one
+from __future__ import absolute_import
```

Comment:
I'm not sure why this is required. It makes no difference with our without it as far as I can tell.

Reply:
I've no idea why I added it either - I can't see any difference with and without. I shall remove it.